http_handlers.go

```diff
@@ -567,22 +599,32 @@ func handlePacksDownload(c echo.Context) error {
 // This is an internal function
 func handlePackDownloadRequest(c echo.Context) error {
 	var (
-		args PackDownloadRequestArgs
-		app  = c.Get("app").(*App)
-		err  error
+		args         PackDownloadRequestArgs
+		app          = c.Get("app").(*App)
+		err          error
+		pack         *Pack
+		packVersion  *PackVersion
+		packFilePath string
 	)
 
 	if err := c.Bind(&args); err != nil {
 		app.log.Printf("error reading request, err: %s", err.Error())
 		return echo.NewHTTPError(http.StatusBadRequest, fmt.Sprintf("error getting metadata. message: %s", err.Error()))
 	}
 
-	packFilePath, err := downloadPackFile(args.LangCode, args.PackVersionIdentifier)
+	pack, packVersion, packFilePath, err = downloadPackFile(args.LangCode, args.PackIdentifier, args.PackVersionIdentifier)
 	if err != nil {
 		return echo.NewHTTPError(http.StatusBadRequest, fmt.Sprintf("error downloading pack: %s", err.Error()))
 	}
 
-	learnWordsFromFile(c, args.LangCode, packFilePath)
+	// Learn from pack file and don't remove it
+	learnWordsFromFile(c, args.LangCode, packFilePath, false)
```

Comment:
can unparam boolean parameter for deletion.

Reply:
@joicemjoseph Sorry, but I don't understand what you meant by that.

Reply:
you can remove last `false` in `learnWordsFromFile`

Reply:
But that gives the error `not enough arguments in call to learnWordsFromFile`

Reply:
I mean remove it from `learnWordsFromFile` function

Reply:
Pack files need to be retained and not deleted, so that it can be reused later for sharing. In a LAN setup, one host can pull the packs, the rest of the devices in LAN can then pull from this host in LAN.